TM1py/Utils/Utils.py

```diff
@@ -533,6 +535,23 @@ def get_seconds_from_duration(time_str: str) -> int:
     seconds = (int(d) * 86400) + (int(h) * 3600) + (int(m) * 60) + int(s)
     return seconds
 
+def url_parameters_add(url, **kwargs: str) -> str:
```

Comment:
I think it may a good investment to add some unit tests for this function.

As a reference, you can check out the `format_url` tests in the MdxUtils Tests:
https://github.com/cubewise-code/tm1py/blob/master/Tests/MDXUtils.py